lib/sql.js

```diff
@@ -690,9 +690,24 @@ SQLConnector.prototype.update = function(model, where, data, options, cb) {
  */
 SQLConnector.prototype._replace = function(model, where, data, options, cb) {
   var stmt = this.buildReplace(model, where, data, options);
-  this.execute(stmt.sql, stmt.params, options, cb);
+  this.execute(stmt.sql, stmt.params, options, function(err, info) {
+    if (err) return cb(err);
+    if (info[0]) info = info.shift();
```

Comment:
@bajtos Do you think it might be worth going a step further and return an Error if the data arrives in any format that isn't the expected one?
```
if (!info || !info.affectedRows) {
  return cb(new Error('Invalid format')); // The error msg would need improving, of course.
}
```